mimic_2_data_analysis.ipynb

```diff
@@ -0,0 +1,3433 @@
+{
```

Comment:
<p>Maybe add a comment here that <code>CoxPHFitter</code> requires a pandas DataFrame as input which is why we are creating a pd.DF here and passing it? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='131'/>

Reply:
<p>No problem.</p>